.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,198 @@
+name: Build, test and deploy
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    ~
+
+env:
+  DOCKERHUB_TAG: godebos/debos
+  GITHUB_TAG: ghcr.io/${{ github.repository_owner }}/debos
```

Comment:
I like this form because a downstream fork can use it to push their own images to their own packages on GitHub. We can't make that automatically possible for DockerHub, but I don't see why we want to prevent forks from getting this benefit within Github? What's the downside for us here?

Reply:
does it automatically create the container registry for the fork, or are manual steps required to do that for a fresh fork ?

Reply:
As I understand it, the registry always exists. I didn't do any setup for my fork.

Reply:
nice! In that case, can we replace `debos` with `${{ github.repository }}` in case forks don't use the same name ?